mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -305,13 +305,13 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD ac setOnFireFromLava ()V
 	METHOD ad extinguish ()V
 	METHOD ae destroy ()V
-	METHOD af calculateNextStepDistance ()F
+	METHOD af calculateStepDelta ()F
 	METHOD ag moveToBoundingBoxCenter ()V
 	METHOD ah getSwimSound ()Lyk;
 	METHOD ai getSplashSound ()Lyk;
 	METHOD aj getHighSpeedSplashSound ()Lyk;
 	METHOD ak checkBlockCollision ()V
-	METHOD al isJesus ()Z
+	METHOD al playsFlyingSounds ()Z
```

Comment:
This still doesn't seem right to me. The name sounds too long when I try to read it.

If anyone has any ideas for a better alternative, I'm all ears.